libshaderc_util/include/libshaderc_util/compiler.h

```diff
@@ -76,7 +76,7 @@ class Compiler {
     Vulkan,        // Default to Vulkan 1.0
     OpenGL,        // Default to OpenGL 4.5
     OpenGLCompat,  // Deprecated.
-    WebGPU,
+    WebGPU,        // Deprecated.
```

Comment:
libshaderc_util is actually internal.  Thanks for keeping this, but I *think* would have been ok to remove.

Reply:
I have removed it now